views/templates/public.ejs

```diff
@@ -7,7 +7,7 @@
       		<% if(settingsDatabase.registrationEnabled) { %>
         		<% if(path == '/register'){ %> <%- include("../pages/public/register") %> <% } %> 
       		<% } %>
-      		<!-- /Page Content -->
+      		<!-- Page Content -->
```

Comment:
Please make this say End Page Content to represent this is the ending of this section

Reply:
@Facinorous-420 I reverted that change